source/_integrations/gryfsmart.markdown

```
@@ -0,0 +1,207 @@
+          o_id: 34            # Combined ID: controller 3, pin 4 (for example)
+          t_id: 21            # Combined ID: controller 2, pin 1 (for example)
+    binary_input:
+        - name: "binary"
+          id: 34
+          device_class: door
+```
+
+### 2.1 Configuration Schemes
+
+The configuration for each type follows specific rules. Remember:
+- Names may include uppercase letters, numbers, and spaces, but no special characters other than "_".
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)